core/discovery.go

```diff
@@ -19,9 +19,43 @@ under the License.
 
 package core
 
-import "github.com/spf13/viper"
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
 
-// GetRootNode place holder function for providing a boostrap address for a Validating Peer.
-func GetRootNode() (string, error) {
-	return viper.GetString("peer.discovery.rootnode"), nil
+	"github.com/spf13/viper"
+
+	
+)
+
+type StaticDiscovery struct {
```

Comment:
Please add comments to public types and functions in the form `// StaticDiscovery ...`